client/src/components/Sidebar/NavigationLinks.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import ListItemText from '@material-ui/core/ListItemText'
+import classNames from 'classnames'
+import { makeStyles } from '@material-ui/core/styles'
+import List from '@material-ui/core/List'
+import styles from '../../assets/jss/material-dashboard-react/components/sidebarStyle'
+
+import Icon from '@material-ui/core/Icon'
+import { NavLink } from 'react-router-dom'
+import ListItem from '@material-ui/core/ListItem'
+const useStyles = makeStyles(styles)
+
+export const NavigationLinks = props => {
+  const classes = useStyles()
+  let { links, color, activeRoute } = props
+  return (
+    <List className={classes.list}>
+      {links &&
+        links.map((prop, key) => {
+          let activePro = ' '
+          let listItemClasses
+          if (prop.path === '/upgrade-to-pro') {
+            activePro = classes.activePro + ' '
+            listItemClasses = classNames({
+              [' ' + classes[color]]: true
```

Comment:
I think we need to use template literal here. Is it possible to use template literal?